### PR TITLE
related to #36 : added the .env_sample and instructions in the readme to configure it

### PR DESCRIPTION
**Pull request by psnmissaka**
Relates to #36 
*Added a sample .env file for users to configure the SendGrid api key.
*Included dotenv package to load the environment variables from .env file
*Added instructions in the readme file to setup the environment variables and to configure API key in the .env file

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/nodejs-http-client?pullRequest=46) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/nodejs-http-client?pullRequest=46) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/nodejs-http-client?pullRequest=46) it.</sub>

File: README.md [hunk 1]
```diff
@@ -31,6 +31,16 @@ All updates to this library are documented in our [CHANGELOG](https://github.com
 
 - Node.js version 0.10, 0.12 or 4
 
+You need to have a API key to use the app. To configure the API key in the environment variables follow these steps.
+ - create .env file in the root directory of the project.
+ - Copy the contents of .env_sample file in to the .env file
+ - Once you have obtained the API key, find the environment variable SENDGRID_API_KEY and assign your API key as it's value.
+ As an example it should look like this
+ ```
+ SENDGRID_API_KEY='Ef34Gyt&dfPFC34G';
```
Comment (mbernier) [inline README.md]: Can you please change this to a placeholder like so:
```
SENDGRID_API_KEY='<YOUR_API_KEY>';
```

Comment (psnmissaka) [reply README.md]: Changed the placeholder as requested with the latest commit

**Comment by mbernier** [comment]
This does exactly what we were looking for. Thank you! 

In this case, it's a good thing to have a broken build!

**Comment by thinkingserious** [comment]
Hello @psnmissaka,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by psnmissaka** [comment]
you are welcome @thinkingserious i'm glad to be able to contribute. I just filled the form as well.

**Pull request merged**